vlib/v/parser/tests/interface_mutability_receiver.out

```diff
@@ -0,0 +1,8 @@
+vlib/v/parser/tests/interface_mutability_receiver.vv:23:18: error: `Dog` incorrectly implements method `set_name` of interface `Animal`: expected `Animal` which is immutable, not `mut &Dog`
+   21 |     dog := Dog{'Doggo'}
+   22 |     println(dog.name)
+   23 |     set_animal_name(dog, 'Pupper')
+      |                     ~~~
+   24 |     println(dog.name)
+   25 | }
+Details: main.Animal has `set_name(name string)`
```

Comment:
This error is confusing to me. Can you add an additional detail, that prints the signature `fn (mut d Dog) set_name(name string)` too, so that it can be clearer to the difference between the expected method from the interface, and given method?

Reply:
good idea

Reply:
added more detail: waiting for tests